composer.json

```diff
@@ -14,7 +14,7 @@
     "minimum-stability": "dev",
     "require": {
         "php": "^8.3",
-        "league/csv": "^9.18",
+        "league/csv": "^9.27",
```

Comment:
Needed to get the new API that replaces the deprecated API so that the --prefer-lowest build does not fail - https://github.com/thephpleague/csv/releases/tag/9.27.0